jax/_src/distributed.py

```diff
@@ -179,7 +190,9 @@ def initialize(coordinator_address: str | None = None,
   ``cluster_detection_method="mpi4py"`` to bootstrap the required arguments.
 
   Otherwise, you must provide the ``coordinator_address``,
-  ``num_processes``, and ``process_id`` arguments to :func:`~jax.distributed.initialize`.
+  ``num_processes``, ``process_id``, and ``local_device_ids`` arguments
+  to :func:`~jax.distributed.initialize`. When all four arguments are provided, cluster
+  envs auto detection will be skipped.
```

Comment:
Nit: expand the abbreviation.
environment auto detection will be skipped.

Reply:
Done.